configs/deepfashion/mask_rcnn_r50_fpn_1x_deepfashion.py

```diff
@@ -0,0 +1,11 @@
+_base_ = [
+    '../_base_/models/mask_rcnn_r50_fpn.py',
+    '../_base_/datasets/deepfashion.py', '../_base_/schedules/schedule_1x.py',
+    '../_base_/default_runtime.py'
+]
+model = dict(
+    roi_head=dict(
+        bbox_head=dict(num_classes=15), mask_head=dict(num_classes=15)))
+# runtime settings
+total_epochs = 12
```

Comment:
This is the default setting? It may be omitted from this config.

Reply:
Delete the unnecessary settings. 
roi_head is different due to different num_classes.

Reply:
It seems that you changed total_epochs from 12 to 15.

Reply:
Yes. 15 epochs is better-performed, so I updated the config, checkpoint and the log.

Reply:
If so, you may update the filename from `1x` to `15e`.

Reply:
Renamed. Thanks for your advice.